weasel/cli/assets.py

```diff
@@ -78,7 +78,7 @@ def project_assets(
         checksum = asset.get("checksum")
         if "git" in asset:
             git_err = (
-                "Cloning spaCy project templates requires Git and the 'git' command. "
+                "Cloning Weasel project templates requires Git and the 'git' command. "
```

Comment:
What's the official spelling, "Weasel" or "weasel"?

Reply:
Good point. I used Weasel/`weasel` v. spaCy/`spacy` but maybe that's not what @svlandeg had in mind?

Reply:
(the README uses weasel, so I can make an educated guess)

Reply:
I'm happy with Weasel/`weasel` !